[2025-05-16T14:40:54.532+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cdc_pipeline.check_consumer manual__2025-05-16T14:40:44.810615+00:00 [queued]>
[2025-05-16T14:40:54.630+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cdc_pipeline.check_consumer manual__2025-05-16T14:40:44.810615+00:00 [queued]>
[2025-05-16T14:40:54.632+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 4
[2025-05-16T14:40:54.736+0000] {taskinstance.py:1380} INFO - Executing <Task(BashOperator): check_consumer> on 2025-05-16 14:40:44.810615+00:00
[2025-05-16T14:40:54.820+0000] {standard_task_runner.py:57} INFO - Started process 3161 to run task
[2025-05-16T14:40:54.827+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'cdc_pipeline', 'check_consumer', 'manual__2025-05-16T14:40:44.810615+00:00', '--job-id', '115', '--raw', '--subdir', 'DAGS_FOLDER/cdc_pipeline_dag.py', '--cfg-path', '/tmp/tmpthmpyvv4']
[2025-05-16T14:40:54.834+0000] {standard_task_runner.py:85} INFO - Job 115: Subtask check_consumer
[2025-05-16T14:40:55.242+0000] {task_command.py:415} INFO - Running <TaskInstance: cdc_pipeline.check_consumer manual__2025-05-16T14:40:44.810615+00:00 [running]> on host 2325534d36fc
[2025-05-16T14:40:55.854+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='cdc_pipeline' AIRFLOW_CTX_TASK_ID='check_consumer' AIRFLOW_CTX_EXECUTION_DATE='2025-05-16T14:40:44.810615+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-16T14:40:44.810615+00:00'
[2025-05-16T14:40:55.918+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-05-16T14:40:55.925+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'nc -z kafka-connect 8083 || (echo "Kafka Connect not running!" && exit 1)']
[2025-05-16T14:40:56.028+0000] {subprocess.py:86} INFO - Output:
[2025-05-16T14:40:56.117+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-05-16T14:40:56.357+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=cdc_pipeline, task_id=check_consumer, execution_date=20250516T144044, start_date=20250516T144054, end_date=20250516T144056
[2025-05-16T14:40:56.616+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-05-16T14:40:56.744+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
